### PR TITLE
New: Add more groups that do not follow -Group format

### PR DESCRIPTION
**Pull request by Tamberlox**
#### Database Migration
NO

#### Description
Adds a few more groups that do not follow the conventional -Group format

#### Todos
/

#### Issues Fixed or Closed by this PR

/

### CODE REVIEW

**Comment by bakerboy448** [comment]
Tests fail

> Expected string to be "Ghost" with a length of 5, but "QxR" has a length of 3, differs near "QxR" (index 0).

File: src/NzbDrone.Core/Parser/Parser.cs [hunk 1]
```diff
@@ -457,7 +457,7 @@ public static class Parser
         private static readonly Regex ExceptionReleaseGroupRegexExact = new Regex(@"(?<releasegroup>(?:D\-Z0N3|Fight-BB)\b)", RegexOptions.IgnoreCase | RegexOptions.Compiled);
 
         //groups whose releases end with RlsGroup) or RlsGroup]
-        private static readonly Regex ExceptionReleaseGroupRegex = new Regex(@"(?<releasegroup>(Tigole|Joy|ImE|UTR|t3nzin|Anime Time|Project Angel|Hakata Ramen)(?=\]|\)))", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex ExceptionReleaseGroupRegex = new Regex(@"(?<releasegroup>(QxR|Silence|afm72|Panda|Ghost|MONOLITH|Tigole|Joy|ImE|UTR|t3nzin|Anime Time|Project Angel|Hakata Ramen)(?=\]|\)))", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Parser/Parser.cs:460]: anything with QXR will be detected - are they the actual group? it seems they're the organization and then the various other groups are part of it?

QxR being here also causes the tests containing QxR and not expecting UxR to fail

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

File: src/NzbDrone.Core/Parser/Parser.cs (hunk 1 above)
Comment (Tamberlox) [reply src/NzbDrone.Core/Parser/Parser.cs:460]: It's the same case as with UTR actually. I'm not sure how you guys managed that use case though.

**Comment by Tamberlox** [comment]
I've gone and removed QxR from the code.

**Pull request merged**